src/python/WMCore/WMRuntime/Watchdog.py

```diff
@@ -78,6 +81,37 @@ def setupMonitors(self, task, wmbsJob):
                 #This should be a config section
                 monitorArgs = getattr(task.data.watchdog, monitor)
                 args = monitorArgs.dictionary_()
+                # Scale resources according to the HTCondor runtime environment.
+                origCores = 1
+                try:
+                    origCores = int(task.data.application.multicore.numberOfCores)
+                except AttributeError:
+                    print("No value for numberOfCores.  Assuming 1")
+                resources = {'cores': origCores}
+                origMaxRSS = args.get('maxRSS')
+                if origMaxRSS:
+                    resources['memory'] = origMaxRSS
+                # Actually parses the HTCondor runtime
+                resizeResources(resources)
+                # HTCondor doesn't explicitly scale VSize; hence we scale it
+                # according to the scaling of maxRSS between the workflow and runtime.
+                if origMaxRSS and ('maxVSize' in args):
+                    args['maxVSize'] *= resources['memory'] / origMaxRSS
```

Comment:
For now, I think I'll set:  "If we change max RSS for resizable jobs, disable max VSize monitoring".
- There's no clear reason why max VSize should scale with the number of CPUs.  If max vsize is 4GB for 4 cores, it is not necessarily 1GB for 1 core.
- The only alternate I can think of is take the maximum RSS and add a flat number (i.e., 2GB).

Just removing the cap feels less hacky though.